client/app/states/requests/list/list.state.js

```diff
@@ -43,7 +43,7 @@
   }
 
   /** @ngInject */
-  function StateController($state, requests, orders, RequestsState, $filter, $rootScope, lodash) {
+  function StateController($state, requests, orders, RequestsState, OrdersState, $filter, $rootScope, lodash) {
```

Comment:
Well, I'm kinda hoping to "solve" this by not having the test fail when that happens :).. 

As for making the list smaller, well, you *could* write a factory that gets injected a bunch of dependencies and returns them wrapped in an object.. I wonder how useful that'd be, maybe it does make sense for commonly injected combinations..